applications/quench_processing/quench_cryomodule.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+import sys
+
+from lcls_tools.common.logger import logger
+
+from applications.quench_processing.quench_cavity import QuenchCavity
+from utils.sc_linac.cryomodule import Cryomodule
+from utils.sc_linac.linac import Machine
+
+
+class QuenchCryomodule(Cryomodule):
+    def __init__(self, cryo_name, linac_object):
+        super().__init__(cryo_name, linac_object)
+
+        formatter = logging.Formatter(
+            fmt="%(asctime)s %(levelname)-8s %(message)s",
+            datefmt="%Y-%m-%d %H:%M:%S",
+        )
+
+        self.console_handler = logging.StreamHandler(sys.stdout)
+        self.console_handler.setFormatter(formatter)
+
+        self.logger = logger.custom_logger(f"{self} quench resetter")
+        self.logger.setLevel(logging.DEBUG)
+
+        self.logfile = f"logfiles/cm{self.name}/cm{self.name}_quench_reset.log"
+        os.makedirs(os.path.dirname(self.logfile), exist_ok=True)
+
+        self.file_handler = logging.FileHandler(self.logfile, mode="a")
```

Comment:
I noticed the mode got changed from "w" to "a." Write vs append. Intentional?

Reply:
Ya I don't know why I was overwriting the log files before. That kind of defeats the purpose of a log lol